app/queries/distribution_summary_by_county_query.rb

```diff
@@ -0,0 +1,78 @@
+class DistributionSummaryByCountyQuery
```

Comment:
Is there a reason this needs to be an instance instead of just having a class function? It's easier to test, stub and work with.

Reply:
Also can we return a data class instead of a hash?

Reply:
> Is there a reason this needs to be an instance instead of just having a class function? It's easier to test, stub and work with.

No real reason, current queries in this folder vary (sometimes `call` is an instance method sometimes a class method). 

Changed it to be a class method.

Reply:
> Also can we return a data class instead of a hash?

Fixed. Out of curiosity, why do you prefer data objects rather than hashes?